### DISCUSSION TITLE
acess to chirp main file

### DISCUSSION THREAD

**Discussion by joc2** [egzumer/uvk5-chirp-driver#26] [General]
why a can't update file chirp anymore ????   protected main branch....

**Comment by OE1MWW** [comment]
can you please once more explain, what you a trying to do ?
Do you want to copy the 'CHIRP_egzumer.exe ' - ?
What Windows are you using ?

**Reply by joc2** [reply]
No ,no
i can not commit modification direct in file uvk5_egzumer.py and be online few second after... has it was yesterday,  
I fix thing, match with radio menu etc... 
yesterday it was working but today not.
because when i summit the change this morning and i got protect main branch !!!! 
that why i asking egzumer if something change ....???

**Comment by egzumer** [comment]
As we talked, the driver is in a state at which I don't want more changes. Main branch is locked, you can commit to side branches as you already did. You can also create pull requests with GH (preferred way).
